src/index.html

```diff
@@ -17,16 +17,20 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form action="post">
```

Comment:
The action attribute of the form element should be a valid URL. If you intend to use a POST method, make sure to specify it correctly with `method="POST"`.